### PR TITLE
Make suite.go more consistent

### PR DESCRIPTION
**Pull request by sebrandon1**
The diff makes this look more complicated than it is.  All I am doing is moving the:

```
testID := identifiers.XformToGinkgoItIdentifier(identifiers.XXX)
	ginkgo.It(testID, ginkgo.Label(Online, testID), func() {
```

To wrap the calling functions like it is done in other test suites.  I also moved the `testhelper` skip checks out of the func if applicable.

### CODE REVIEW

File: cnf-certification-test/operator/suite.go [hunk 1]
```diff
@@ -42,24 +42,22 @@ var _ = ginkgo.Describe(common.OperatorTestKey, func() {
 	var env provider.TestEnvironment
 	ginkgo.BeforeEach(func() {
 		env = provider.GetTestEnvironment()
+		testhelper.SkipIfEmptyAny(ginkgo.Skip, env.Operators)
```
Comment (edcdavid) [inline cnf-certification-test/operator/suite.go:45]: I would revert the change here and add this under each it statement. Otherwise, just modifying one TC will require to check all others (when adding new TCs or modifying). It is way simpler to make the skip relevent to a single TC always.

Comment (sebrandon1) [reply cnf-certification-test/operator/suite.go:45]: Yeah I was 50/50 about moving this originally.  Sounds good I'll switch it back.

Comment (sebrandon1) [reply cnf-certification-test/operator/suite.go]: Fixed!

**Review by edcdavid — APPROVED** [review]

**Review by hamadise — APPROVED** [review]
